akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/TestProbeImpl.scala

```diff
@@ -170,8 +176,7 @@ private[akka] final class TestProbeImpl[M](name: String, system: ActorSystem[_])
   override def receiveMessage(max: FiniteDuration): M = receiveMessage_internal(max.dilated)
 
   def receiveMessage_internal(max: FiniteDuration): M =
-    receiveOne_internal(max).
-      getOrElse(assertFail(s"Timeout ($max) during receiveMessage while waiting for message."))
+    receiveOne_internal(max).getOrElse(assertFail(s"Timeout ($max) during receiveMessage while waiting for message."))
```

Comment:
?

Reply:
it's 119 chars, so fits on one line

(I would have expected it to preserve existing line breaks though)

Reply:
now it's not touching this, not sure what I changed, perhaps the version